bridge_adaptivity/module/forms.py

```diff
@@ -153,9 +154,16 @@ class Meta:
             'grading_policy_name',
             'strict_forward',
             'ui_option',
-            'ui_next'
+            'ui_next',
+            'congratulation_message',
         )
-        labels = {'ui_option': _('UI Option'), 'ui_next': _('Additional NEXT Button')}
+        labels = {
+            'ui_option': _('UI Option'),
+            'ui_next': _('Additional NEXT Button'),
+            'congratulation_message': _('Show congratulation message if score is {}%+').format(
+                settings.CONGRATULATION_SCORE_LEVEL * 100
```

Comment:
What about our experience with the math operations over float numbers? The result could be unpleasantly long.

Reply:
Actually, if the instructor cannot handle the congrats treashold I would avoid any certain numbers in the message. Or cast it to int type.

Reply:
You are right! I forgot the problem related to the math operations over float numbers. I transformed it to the int type.